src/Statistics.jl

```diff
@@ -222,15 +220,15 @@ function mean!(
         j = order[i]
         s += w[j]
         t = w[j] / s
-        log!(M, v, y, x[j])
-        exp!(M, ytmp, y, v, t)
+        inverse_retract!(M, v, y, x[j], inverse_retraction)
+        retract!(M, ytmp, y, v, t, retraction)
         copyto!(y, ytmp)
     end
     return y
 end
 
 @doc doc"""
-    median(M::Manifold, x::AbstractVector, [w::AbstractWeights]; kwargs...)
+    median(M::Manifold, x::AbstractVector [, w::AbstractWeights]; kwargs...)
```

Comment:
this is really just a minor remark, but here you have `[, ... ]` while in 244 you have `[ ... ,]` - is that intentional or would it maybe be cool to have that unified? In general I like the `[]`-notation.

Reply:
I ended up unifying under the `[, ... ]` notation. I think it's the most sensible in most cases, the exception being in multiline signatures. It's also the syntax the Julia docs use. I prefer the comma inside because if the argument isn't included in the signature, then the comma makes no sense.